common.blocks/sidebar/__sorting/sidebar__sorting.js

```diff
@@ -25,13 +25,16 @@ modules.define(
             window.location.reload();
         },
 
-        _getParams(sort, direction) {
-            const uri = Location.getUri();
-            const query = uri.queryParams;
-            const params = {};
+        _getParams: function(sort, direction) {
+            var uri = Location.getUri();
+            var query = uri.queryParams;
+            var params = {};
+            var exclude_params = ['pagination'];
```

Comment:
1. camelCase
2. if there's just one param to exclude there's no need in array and if want to make code good for future replace an array to hash to reduce complexity from `O(n)` to `O(1)`:
```js
var excludeParams = {
    pagination: true
};

excludeParams[key] || (params[key] = query[key]);
```
3. I believe there's still no reason to have that `forEach` bellow as it should be enough to add `{ pagination: undefined }` as last argument to `Object.assign`

BTW, there are still at least 2 use cases of ES6 in this file ;)